modules/Microsoft.HealthcareApis/workspaces/iotconnectors/readme.md

```diff
@@ -0,0 +1,308 @@
+        }
+    }
+}
+```
+
+</details>
+
+<details>
+
+<summary>Bicep format</summary>
+
+```bicep
+destinationMapping: {
+    templateType: 'CodeValueFhir'
+    template: {
+        codes: [
+            {
+                code: '8867-4'
+                system: 'http://loinc.org'
+                display: 'Heart rate'
+            }
+        ],
+        periodInterval: 60,
+        typeName: 'heartrate'
+        value: {
+            defaultPeriod: 5000
+            unit: 'count/min'
+            valueName: 'hr'
+            valueType: 'SampledData'
+        }
+    }
+}
+```
+
+</details>
+
+<p>
+
+### Parameter Usage: `tags`
+
+Tag names and tag values can be provided as needed. A tag can be left without a value.
+
+<details>
+
+<summary>Parameter JSON format</summary>
+
+```json
+"tags": {
+    "value": {
+        "Environment": "Non-Prod",
+        "Contact": "test.user@testcompany.com",
+        "PurchaseOrder": "1234",
+        "CostCenter": "7890",
+        "ServiceName": "DeploymentValidation",
+        "Role": "DeploymentValidation"
+    }
+}
+```
+
+</details>
+
+<details>
+
+<summary>Bicep format</summary>
+
+```bicep
+tags: {
+    Environment: 'Non-Prod'
+    Contact: 'test.user@testcompany.com'
+    PurchaseOrder: '1234'
+    CostCenter: '7890'
+    ServiceName: 'DeploymentValidation'
+    Role: 'DeploymentValidation'
+}
+```
+
+</details>
+<p>
+
+### Parameter Usage: `userAssignedIdentities`
+
+You can specify multiple user assigned identities to a resource by providing additional resource IDs using the following format:
+
+<details>
+
+<summary>Parameter JSON format</summary>
+
+```json
+"userAssignedIdentities": {
+    "value": {
+        "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-sxx-az-msi-x-001": {},
+        "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-sxx-az-msi-x-002": {}
+    }
+}
+```
+
+</details>
+
+<details>
+
+<summary>Bicep format</summary>
+
+```bicep
+userAssignedIdentities: {
+    '/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-sxx-az-msi-x-001': {}
+    '/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-sxx-az-msi-x-002': {}
+}
+```
+
+</details>
+<p>
+
+## Outputs
+
+| Output Name | Type | Description |
+| :-- | :-- | :-- |
+| `location` | string | The location the resource was deployed into. |
+| `name` | string | The name of the medtech service. |
+| `resourceGroupName` | string | The resource group where the namespace is deployed. |
+| `resourceId` | string | The resource ID of the medtech service. |
+| `systemAssignedPrincipalId` | string | The principal ID of the system assigned identity. |
+| `workspaceName` | string | The name of the medtech workspace. |
+
+## Deployment examples
```

Comment:
I appreciate that you added the deployment examples here - however - the Set-ModuleReadMe script actually autogenerates them for the top-level readme. If you're wondering why only there - the answer is because we didn't implement a logic yet to extract an deployment example out of the top-level test files. 
In this case I'd actually ask you to remove them, be it simply for the reason that there is a fair chance this example will run stale if it's not generated from actual deployment code.
 
However, I'm happy to hear your thoughts about that :)

Reply:
PS: The same is by the way true for the `Parameter Usage` and there is actually an open issue to also extract them from code. For the time being we 'allow' them to run stale if fix them if noticed.